docs/config.rst

```diff
@@ -163,9 +158,9 @@ Upgrading ``astropy``
 Each time you upgrade to a new major version of ``astropy``, the
 configuration parameters may have changed.
 
-If you never edited your configuration file, there is nothing for you
-to do. It will automatically be replaced with a configuration file
-template for the newly installed version of ``astropy``.
+If you never edited your configuration file, there is nothing for you to do. It
+will automatically be replaced with a configuration file template for the newly
```

Comment:
"automatically" is also misleading now, as generating a new `astropy.cfg` requires manual intervention.